packages/story-editor/src/app/media/utils/useFFmpeg.js

```diff
@@ -457,6 +457,59 @@ function useFFmpeg() {
     [getFFmpegInstance]
   );
 
+  /**
+   * Converts any audio file format to MP3 using FFmpeg.
+   *
+   * @param {File} file Original audio file object.
+   * @return {Promise<File>} Converted video file object.
+   */
+  const convertToMp3 = useCallback(
+    async (file) => {
+      //eslint-disable-next-line @wordpress/no-unused-vars-before-return -- False positive because of the finally().
+      const trackTiming = getTimeTracker('load_mp3_conversion');
+
+      let ffmpeg;
+
+      try {
+        ffmpeg = await getFFmpegInstance(file);
+
+        const tempFileName = uuidv4() + '.mp3';
```

Comment:
Can we use `AUDIO_FILE_TYPE`?

Reply:
Ditto https://github.com/GoogleForCreators/web-stories-wp/pull/11887#discussion_r918123364